pkg/apparmor/apparmor_linux.go

```diff
@@ -32,6 +32,17 @@ func IsEnabled() bool {
 	return supported.NewAppArmorVerifier().IsSupported() == nil
 }
 
+// IsAppArmorAvailable returns true if the `apparmor_parser` binary is found on the host,
+// which is required to apply profiles. This function was added in parallel to IsEnabled
+// to avoid breaking existing users of IsEnabled. Prefer using IsAppArmorAvailable for
+// new implementations.
+func IsAppArmorAvailable() bool {
```

Comment:
Doesn't this trade one problem for another? i.e. just because the binary exist doesn't mean the system supports or has apparmor enabled? One easy example would be in a container.

It is not clear to me that using this for cri-o would be any better.

Reply:
Well, while I agree that only checking for the binary is not enough and perhaps something [like this](https://github.com/opencontainers/runc/blob/bf9e609b3ba77a9e01fb96071133c99d4167ee88/libcontainer/apparmor/apparmor_linux.go#L18) would be better, you already do have [IsEnabled()](https://github.com/containers/common/blob/f01ad5b683b91f031ba4248ea3264260ecad757b/pkg/apparmor/internal/supported/supported.go#L43), that does similar thing.